theano/gpuarray/pool.py

```diff
@@ -411,15 +412,102 @@ def make_node(self, inp, eval_point, ws, stride=None, pad=None):
 
         return Apply(self, [inp, eval_point, ws, stride, pad], [eval_point.type()])
 
+
+class GpuRoIPoolOp(CGpuKernelBase):
+
+    __props__ = ('spatial_scale', 'pooled_h', 'pooled_w')
```

Comment:
Put the props in the same order as the parameter in \__init__. Idem for the second class.